src/Logic/Forms/RemoveTextForHI.cs

```diff
@@ -290,7 +289,7 @@ public string RemoveColon(string text)
                 {
                     if (indexOfDialogChar < 0 || indexOfDialogChar > 4)
                     {
-                        var st = new StripableText(newText, "", "");
+                        var st = new StripableText(newText, string.Empty, string.Empty);

```

Comment:
<code>String.Empty</code> shouldn't be confused as "" which may contain some weird unicode whitespace